text/0000-bee-pow/0000-bee-pow.md

```diff
@@ -0,0 +1,70 @@
++ Feature name: `bee-pow`
++ Start date: 2019-10-03
++ RFC PR: [iotaledger/bee-rfcs#0000](https://github.com/iotaledger/bee-rfcs/pull/0000)
++ Bee issue: [iotaledger/bee#0000](https://github.com/iotaledger/bee/issues/0000)
+
+# Summary
+
+This RFC proposes a dedicated crate to perform proof-of-work (PoW) for a single gossip message, that attempts to become accepted and propagated by the network. 
+
+# Motivation
+
+In order to protect the network from spam each device that wishes to get a message (e.g. a value transaction) propagated by all the nodes needs a way to proove to the network, that it has invested the necessary amount of computational work either by doing the computation by itself, or by externalizing it using some PoW service provider. 
+
+### Background info on PoW
+Cryptographic hash functions are one-way-functions. That means in simple terms that given the output you cannot calculate the input from it. Proof-of-Work works by setting some constraint on the output like a certain number of zeros at the end or the beginning of the hash. It's completely arbitrary. It could be ten `1`s or the sequence `1234567890` because any particular sequence is equally likely. What matters is that your only chance to find an input that satisfies the constraint is guessing/brute-forcing it. You can also easily set the difficulty by making the constraint harder to satisfy (extend the sequence). So to check if PoW was correctly done all a validating node has to do is hash the given message which includes the nonce, and see if the constraint is satisfied. On the other hand the message publisher has to repeat the following cycle many times until it has found a valid nonce:
```

Comment:
This section could be even shorter and instead include a wiki link to PoW,
we did not come up with PoW

Reply:
you're absolutely right. I will add [this link](https://docs.iota.org/docs/dev-essentials/0.1/concepts/minimum-weight-magnitude).

I hoped that by giving it the title "Background info on PoW", that it would be clear, that we didn't come up with it. 

Do you suggest to remove that section altogether and just put the link there, or should it remain in the RFC, just shortened? @tsvisabo